packages/webpack-extensive-lodash-replacement-plugin/index.js

```diff
@@ -1,8 +1,6 @@
 /* eslint-disable import/no-extraneous-dependencies */
 /* eslint-disable import/no-nodejs-modules */
 const path = require( 'path' );
-const findPackageJson = require( 'find-package-json' );
-const fs = require( 'fs' );
 const semver = require( 'semver' );
 
 // List of all known lodash module names.
```

Comment:
Can we move this to a separate file with just the list? Makes the plugin code easier to navigate and read: much less scrolling 🙂

Reply:
Done!